template_config.yml

```diff
@@ -31,7 +31,7 @@ pydocstyle: true
 pypi_username: pulp
 redmine_project: certguard
 stable_branch: null
-test_bindings: true
+test_bindings: false
```

Comment:
Do we want this to be true?

Reply:
it was an old change, `test_bindings.py` was a blank file, 
`test_bindings: true` is creating one job for running a blank file